internal/xds/bootstrap/bootstrap.go

```diff
@@ -86,6 +83,7 @@ type bootstrapParameters struct {
 	// EnablePrometheusCompression defines whether to enable HTTP compression on metrics endpoint for prometheus.
 	EnablePrometheusCompression bool
```

Comment:
should we delete fields no longer used ?

Reply:
I recall I addressed this with a TODO for following refactor.